bsockMiddleware.js

```diff
@@ -24,8 +24,8 @@ export default function bsockMiddleware (options) {
           socket.close();
         }
         // if it has a `connect` action then we connect to the bcoin socket
-        const { port, host, ssl, protocols } = action.bsock;
-        socket = bsock.connect(port, host, ssl, protocols);
+        const { port, host, ssl, protocols, namespace, apiKey } = action.bsock;
```

Comment:
Will the api key will show up in redux devtools if done this way?

Reply:
I suppose that isn't the worst, as the devtools shouldn't be built into the app everywhere, but other plugins could scrape any actions